hyou/view.py

```diff
@@ -22,9 +22,15 @@
 from . import util
 
 
+DEFAULT_MAJOR_DIMENSION = 'ROWS'
+DEFAULT_VALUE_TYPE = 'FORMATTED_VALUE'
+DEFAULT_DATE_TYPE = 'FORMATTED_STRING'
+
+
 class View(util.CustomMutableFixedList):
 
-    def __init__(self, worksheet, api, start_row, end_row, start_col, end_col):
+    def __init__(self, worksheet, api, start_row, end_row,
+                 start_col, end_col, **kwargs):
```

Comment:
What do you think of adding the new arguments directly as keyword arguments?

If so maybe a docstring would also be helpful to understand the new arguments.

```python
def __init__(self, worksheet, api, start_row, end_row, start_col, end_col,
             major_dimension=DEFAULT_MAJOR_DIMENSION, value_type=DEFAULT_VALUE_TYPE,
             date_type=DEFAULT_DATE_TYPE, **kwargs):
```

Reply:
It's definitely clearer, the method signature is somewhat lengthy though... I'll change it.